### PR TITLE
NCMPCPP: new recipe

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

File: media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+fi
+
+PROVIDES="
+	ncmpcpp$secondaryArchSuffix = $portVersion
+	cmd:ncmpcpp
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	boost$secondaryArchSuffix
+	ncurses6$secondaryArchSuffix
```
Comment (korli) [inline media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe]: boost$secondaryArchSuffix and ncurses6$secondaryArchSuffix should be removed or replaced by something else.

File: media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	lib:libreadline$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libtag$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	boost${secondaryArchSuffix}_devel
+	ncurses6${secondaryArchSuffix}_devel
```
Comment (korli) [inline media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe]: boost${secondaryArchSuffix}_devel and ncurses6${secondaryArchSuffix}_devel should be removed or replaced by something else.

File: media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	devel:libcurl$secondaryArchSuffix
+	devel:libfftw3$secondaryArchSuffix
+	devel:libicui18n$secondaryArchSuffix
+	devel:libmpdclient$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libtag$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoreconf$secondaryArchSuffix
```
Comment (korli) [inline media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe]: cmd:autoreconf doesn't need $secondaryArchSuffix

File: media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:ranlib$secondaryArchSuffix
+	cmd:sed
+	cmd:strip
```
Comment (korli) [inline media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe]: cmd:strip does need $secondaryArchSuffix (part of binutils)

File: media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+		--enable-outputs \
+		--enable-visualizer \
+		--enable-clock
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	export PKG_CONFIG_LIBDIR="`finddir B_SYSTEM_DIRECTORY`/$relativeDevelopLibDir/pkgconfig"
```
Comment (korli) [inline media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe]: Shoulnd't be needed. pkg-config is only called by the configure script.

File: media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+	sed -i -e "s,\$(prefix)/share/doc/\$(PACKAGE),$docDir,g" Makefile doc/Makefile
+	sed -i -e "s,\${prefix}/share,$dataDir,g" Makefile doc/Makefile
```
Comment (korli) [inline media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe]: You should consider patching doc/Makefile.am in PATCH().

File: media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+	make install
+
+	strip $binDir/ncmpcpp
```
Comment (korli) [inline media-sound/ncmpcpp/ncmpcpp-0.7.7.recipe]: strip$secondaryArchSuffix

**Comment by miqlas** [comment]
All the requested changes integrated already.

**Pull request merged**